app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<h2>Tasks:</h2>
+<%# <p>Find me in app/views/tasks/index.html.erb</p> %>
+
+<ul>
+<% @tasks.each do |task|%>
+<li>
+
+
+
+
+
+<%= link_to task.name, task_path(task.id), class: "index-name" %>
+
+
+<% if task.completed == false %>
+<%= button_to "Mark Complete", mark_complete_path(task.id), method: :patch, class: "status-button"  %>
+
+<%else%>
+<%= button_to "Mark Incomplete", mark_complete_path(task.id), method: :patch, class: "status-button"   %>
+
+<%end %>
```

Comment:
this view and the logic in it looks good! You may want to mind your indentation though